CMakeLists.txt

```diff
@@ -185,7 +186,11 @@ if(MSVC)
   if (${QT_PORT})
   # qt specific  flags
   else()
+<<<<<<< HEAD
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /GF /GR /W3 /WX /Zc:wchar_t /favor:blend")# Add MSVS specific flags
+=======
```

Comment:
Guys, I would suggest to run the code before pull requesting it in order to avoid such merge artifacts